taxonomy/algolia/constants.py

```diff
@@ -16,6 +16,7 @@
         'searchable(skills.name)',
         'searchable(industry_names)',
         'searchable(b2c_opt_in)'
```

Comment:
@saleem-latif  There is a missing comma here, and b2c_opt_in values are no longer showing up in the index